pkg/tests/common/appstudio.go

```diff
@@ -19,26 +23,35 @@ var (
 var _ = framework.CommonSuiteDescribe("Red Hat App Studio common E2E", func() {
 	defer g.GinkgoRecover()
 	commonController, err := commonCtrl.NewSuiteController()
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	Expect(err).NotTo(HaveOccurred())
 
 	g.Context("Argo CD", func() {
 		for _, component := range AppStudioComponents {
 			g.It(component+" status", func() {
 				componentStatus, err := commonController.GetAppStudioComponentStatus(component, AppStudioComponentsNamespace)
-				gomega.Expect(err).NotTo(gomega.HaveOccurred())
-				gomega.Expect(componentStatus.Health.Status).To(gomega.Equal(health.HealthStatusHealthy))
+				Expect(err).NotTo(HaveOccurred())
+				Expect(componentStatus.Health.Status).To(Equal(health.HealthStatusHealthy))
 
 			})
 		}
 	})
+	defer g.GinkgoRecover()
 
 	g.Context("ClusterTasks:", func() {
-		for _, pipelineName := range AppStudioPipelinesNames {
-			g.It("Check if "+pipelineName+" clustertask is pre-created", func() {
-				p, err := commonController.GetClusterTask(pipelineName, PipelinesNamespace)
-				gomega.Expect(err).NotTo(gomega.HaveOccurred())
-				gomega.Expect(p.Name).To(gomega.Equal(pipelineName))
+		g.It("Check if AppStudio ClusterTasks are precreated", func() {
+			err := wait.PollImmediate(30*time.Second, 10*time.Minute, func() (done bool, err error) {
+				for _, clusterTaskName := range AppStudioClusterTaskNames {
+					clusterTask, err := commonController.GetClusterTask(clusterTaskName, PipelinesNamespace)
+					if err != nil {
+						return false, nil
```

Comment:
Shouldn't we check what kind of error did we get here - if it's really "not-found" or something else? 
Something like this
```go
if errors.IsNotFound(err) {
    return false, nil
}
return false, err
```
errors taken from "k8s.io/apimachinery/pkg/api/errors" import